src/main/java/com/noticemedan/mappr/model/pathfinding/Dijkstra.java

```diff
@@ -22,16 +22,16 @@ public Dijkstra(Network n, PathNode s) {
 			if (e.getWeight() < 0) throw new IllegalArgumentException("Weight less than zero");
 		});
 
-		distTo = new double[n.getAllNodes().length()];
-		edgeTo = new PathEdge[n.getAllNodes().length()];
+		distTo = new double[n.getAllNodes().size()];
+		edgeTo = new PathEdge[n.getAllNodes().size()];
```

Comment:
Kan personligt godt lide at man skriver objektet ud, så man skriver 'node' eller 'edge' istedet for n eller e. Det gør det nemmere for udefrakommende at læse.

Reply:
Okay n står for 'network' ikke 'node', men hvorfor står s for pathnode?

Reply:
Men det er måske bare bogens konvention?

Reply:
s er for "start", da det er start-noden i pathfinding.
Ja, det er alg4's konventioner.